model/src/test/kotlin/MergedAnalyzerResultTest.kt

```diff
@@ -49,6 +49,19 @@ class MergedAnalyzerResultTest : WordSpec() {
     val analyzerResult2 = AnalyzerResult(true, project2, sortedSetOf(package1, package2, package3), listOf("error-2"))
 
     init {
+        "MergedAnalyzerResult" should {
```

Comment:
Sorry if I'm getting confused here, but should this say "MergedResultsBuilder" (like in the test below), as that's the class being tested?